client/src/components/UpcomingEventsCard/UpcomingEventsCard.css

```diff
@@ -0,0 +1,27 @@
+.UpcomingEventsCard {
+    height: 100%;
+    border-radius: 3px;
+    background-color: var(--pink);
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    color: var(--white);
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    text-decoration: none;
+}
```

Comment:
How much of this is the same as the Card component? Can you eliminate some of this CSS?